### ISSUE TITLE
[Feature Request] configurable automatically hide time

### ISSUE THREAD

**Issue by rebdeg** [jordanbaird/Ice#18]
Could you add a configurable auto hide time. Also a shortcut for temporarily disabling auto hide would be appreciated

**Comment by jordanbaird** [comment]
Yes, that's definitely doable! Right now, autohide works by checking to see when the user clicks outside the menu bar, but I think it would be a good idea to include both options.

**Comment by jordanbaird** [comment]
Just got finished implementing this. As for the custom hotkey, that is something that will likely come later, along with a bunch of additional hotkeys tied to various app features. If you would like, you can open a separate issue for this feature specifically, or I can add it to the roadmap.

**Comment by rebdeg** [comment]
> Just got finished implementing this. As for the custom hotkey, that is something that will likely come later, along with a bunch of additional hotkeys tied to various app features. If you would like, you can open a separate issue for this feature specifically, or I can add it to the roadmap.

Easiest option woud be adding it to the roadmap.
